CHANGELOG.md

```diff
@@ -19,9 +27,6 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Changed
 
-- Rules are now downloaded from the Semgrep Registry in JSON format instead of YAML.
-  This speeds up rule parsing in the Semgrep CLI,
-  making a `semgrep --config auto` run on the semgrep Python package in 14s instead of 16s.
 - Gitlab SAST output is now v14.1.2 compliant
```

Comment:
```suggestion
- Rules are now downloaded from the Semgrep Registry in JSON format instead of YAML.
  This speeds up rule parsing in the Semgrep CLI,
  making a `semgrep --config auto` run on the semgrep Python package in 14s instead of 16s.`
``